app/Libraries/Groups.php

```diff
@@ -70,9 +70,10 @@ public function byIdentifier(string $id): Group
         if ($group === null) {
             try {
                 $group = Group::create([
-                    'identifier' => $id,
+                    'group_desc' => '',
                     'group_name' => $id,
-                    'group_desc' => $id,
+                    'group_type' => 0,
```

Comment:
should be 2 instead ([`GROUP_HIDDEN`](https://github.com/phpbb/phpbb/blob/eb1edd12a1bc6adbb496d043fd2c883f366f452c/phpBB/includes/constants.php#L78-L83))